airflow/providers/http/hooks/http.py

```diff
@@ -258,7 +258,8 @@ def run_with_advanced_retry(self, _retry_args: dict[Any, Any], *args: Any, **kwa
         """
         self._retry_obj = tenacity.Retrying(**_retry_args)
 
-        return self._retry_obj(self.run, *args, **kwargs)
+        # TODO: remove ignore type when https://github.com/jd/tenacity/issues/428 is resolved
+        return self._retry_obj(self.run, *args, **kwargs)  # type: ignore
```

Comment:
I propose merging this PR with this comment, and once the issue is resolved, we can remove the ignore comment. WDYT?

Reply:
Yes